build/lib/electron.ts

```diff
@@ -91,7 +91,7 @@ function darwinBundleDocumentTypes(types: { [name: string]: string | string[] },
 }
 
 export const config = {
-	version: util.getElectronVersion(),
+	version: product.electronRepository ? '19.1.10' : util.getElectronVersion(),
```

Comment:
This is a custom version not available with OSS electron, hence this check. I was thinking of introducing a new field `product.msElectronVersion` in distro but that would require updating distro everytime an Electron version bump is made, even for those where the versions align with OSS Electron which is not ideal in terms of PR flow. I feel this is simple enough for now.